<!--xml-->
<launch>
  <node pkg="my_planner" type="my_planner_node1" respawn="false" name="my_planner" output="screen">
    <rosparam file="$(find my_planner)/launch/waypoints.yaml" command="load" />
    <param name="global_frame" type="string" value="map" /> 
    <param name="robot_base_frame" type="string" value="base_link" /> 
    <param name="transform_tolerance" type="double" value="0.8" /> 
    <param name="control_frequency" type="double" value="10.0" /> 
    <param name="cmd_pub_topic" type="string" value="cmd_vel" />       
  </node>
  

</launch>
